### ISSUE TITLE
Addin manager [more info link] fails

### ISSUE THREAD

**Issue by joemclain** [RickStrahl/MarkdownMonster#846]
MM version 2

The "more info" link on the add-ins displayed in the Addin Manager fail to resolve the destination link correctly and a GitHub 404 page pops up.

Specifically, each "more info" link has a "master" at the end of the link, presumably to access the master branch of the repository. GitHub does not (or no longer) recognizes this. With the "master" part of the url removed the destination resolves correctly.

-joe mclain-

**Comment by RickStrahl** [comment]
This is due to changes in GitHub Urls. It's not actually a problem in MM, but in the addin registry URLs.

Unfortunately the old URLs no longer work - not only that, some of the assumptions made - `master` branch - are no longer good enough to find addins and dependencies.

I've made some changes due to this that will break existing v2 MM installs but fix with the latest update:

* Changed all Addin Repo Urls to point at `/tree/<branch>` so we know what branch to use
* This unfortunately breaks the old addin parser as it can't work with the `/tree`  subpath
* Updated new version to strip out the `/tree` and handle repo to raw Url conversions

Bottom line this is a breaking change that breaks existing installs of v2 for addins. But this change has to be made due to the changes that GitHub put in. Basically the old URLs no longer worked.

Fixes and updates are in 2.3.17+
